ui/pages/settings/index.scss

```
@@ -11,22 +11,138 @@
   flex-flow: column nowrap;
 
   &__header {
-    display: flex;
-    flex-flow: row nowrap;
-    padding: 12px 24px;
-    align-items: center;
-    flex: 0 0 auto;
+    padding: 12px 24px 5px 24px;
+    position: relative;
 
-    &__title {
-      @include H3;
+    @media screen and (max-width: $break-small) {
+      background: #f2f3f4;
+    }
 
-      flex: 1 0 auto;
+    &__title-container {
+      display: flex;
+      flex-flow: row nowrap;
+      align-items: center;
+      flex: 0 0 auto;
+
+      &__close-button {
+        margin-left: auto;
+      }
+
+      &__close-button::after {
+        content: '\00D7';
+        font-size: 40px;
+        color: --ui-4;
+        cursor: pointer;
+      }
+
+      &__title {
+        @include H3;
+
+        flex: 1 0 auto;
+
+        @media screen and (max-width: $break-small) {
+          text-overflow: ellipsis;
+          overflow: hidden;
+          white-space: nowrap;
+          max-width: 250px;
+        }
+      }
+    }
+
+    &__search {
+      padding-bottom: 5px;
+      height: 50px;
 
       @media screen and (max-width: $break-small) {
-        text-overflow: ellipsis;
-        overflow: hidden;
-        white-space: nowrap;
-        max-width: 250px;
+        position: relative;
+      }
+
+      @media screen and (min-width: $break-large) {
+        position: absolute;
+        right: 57px;
+        top: 13px;
+        width: 300px;
+      }
+
+      @media screen and (min-width: $break-midpoint) {
+        width: 400px;
+      }
+
+      &__list {
+        background: #fff;
```

Use css variables we should not be using any static hex / color values in the code base

```suggestion
        background: var(--white);
```

---

Done